### PR TITLE
New sample use case specific playbooks added

### PR DESCRIPTION
**Pull request by cs-shivaji-kolape**
Added new playbooks to **08 - SOC Utilities** playbook collection.
**Get Pending Manual Input**
       Gets all manual input records pending for user action/inputs and create CSV attachments for associated alerts

**Delete Pending Manual Input**
Delete all manual input pending for user action/inputs that have been created before the specified timestamp.

**Terminate Awaiting Data Ingestion playbooks**
Terminates list of data ingestion playbooks which are stuck in awaiting state from last x minutes

UTC
----
Verified playbook working flow with required input parameters

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -14,14 +14,13 @@
...
+    "publisher": "Fortinet",
     "description": "SOC Utilities Solution Pack is designed around utilities to aid in the use of FortiSOAR.",
-    "certified": "false",
-    "help": "https://github.com/fortinet-fortisoar/solution-pack-soc-utilities/blob/release/1.0.0/README.md",
+    "help": null,
     "category": [
         "Utilities"
     ],
-    "supportInfo": null,
+    "supportInfo": "Fortinet Customer Support",
```
Comment (cs-suyog-jinturkar) [inline info.json]: this is uncertified, hence this field value will be empty

File: info.json [hunk 2]
```diff
@@ -14,14 +14,13 @@
         }
     ],
     "prerequisite": "",
-    "publisher": "Fortinet CSE",
+    "publisher": "Fortinet",
```
Comment (cs-suyog-jinturkar) [inline info.json]: It will be marked as Fortinet CSE

File: info.json [hunk 3]
```diff
@@ -16,8 +16,7 @@
     "prerequisite": "",
     "publisher": "Fortinet CSE",
     "description": "SOC Utilities Solution Pack is designed around utilities to aid in the use of FortiSOAR.",
-    "certified": "false",
-    "help": "https://github.com/fortinet-fortisoar/solution-pack-soc-utilities/blob/release/1.0.0/README.md",
+    "help": null,
```
Comment (cs-rohit-khune) [inline info.json:19]: Please add help link in info.json

**Comment by cs-rohit-khune** [comment]
@cs-shivaji-kolape Make sure to update usage.md and contents md for newly added playbooks

**Comment by cs-shivaji-kolape** [comment]
PR incorrectly raise to develop branch

**Pull request closed without merging**